src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Services/DataSetService.cs

```diff
@@ -260,7 +260,8 @@ private async Task LoadMeta(
             dataSetVersion.LocationMetas = await publicDataDbContext.LocationMetas
                 .AsNoTracking()
                 .Where(lm => lm.DataSetVersionId == dataSetVersion.Id)
-                .Include(lm => lm.Options)
+                .Include(lm => lm.OptionLinks)
+                .ThenInclude(lm => lm.Option)
```

Comment:
Not a big one but should this be `loml =>` or even `ml =>` rather than `lm =>`?

Reply:
I've gone with just `l` (as in 'link'). `loml` tends to be too long and ends up being flagged as typos by the IDE dictionary.